tests/lib/state.sh

```diff
@@ -90,6 +90,9 @@ restore_snapd_state() {
         restore_snapd_lib
         cp -rf "$SNAPD_STATE_PATH"/snapd-cache/*  /var/cache/snapd
         cp -rf "$SNAPD_STATE_PATH"/boot/* "$boot_path"
+        # Purge all the systemd service units config
```

Comment:
nitpick: at the first sight, it looks like we are removing the unit files themselves. Adding a "overrides" at the end of the comment would make it more clear.